lib/active_merchant/billing/gateway.rb

```diff
@@ -123,8 +123,9 @@ def generate_unique_id
       class_attribute :supported_cardtypes
       self.supported_cardtypes = []
 
+      # This default list of currencies without fractions are from https://en.wikipedia.org/wiki/ISO_4217
```

Comment:
Nitpick: the primary source is https://www.currency-iso.org/en/home/tables/table-a1.html